<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportMetadataSettingsWidget</class>
 <widget class="QWidget" name="ExportMetadataSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Metadata</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="4" column="0" rowspan="1" colspan="1">
       <!-- This should go first to keep size (TODO: Probably this will be moved to rows 4-5 because it may cause problems at too narrow windows) -->
       <spacer name="filterHintSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="motionCheckBox">
        <property name="text">
         <string>Motion</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="objectsCheckBox">
        <property name="text">
         <string>Objects</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="1">
       <widget class="QLabel" name="objectTypeLabel">
        <property name="text">
         <string>Object type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>objectTypesButton</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="1">
       <widget class="nx::vms::client::desktop::SelectAnalyticsObjectTypesButton" name="objectTypesButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="1">
       <widget class="QLabel" name="hasAttributesLabel">
        <property name="text">
         <string>Has attributes</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>attributeTextEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="1">
       <widget class="QPlainTextEdit" name="attributeTextEdit"/>
      </item>

      <item row="4" column="2" colspan="1">
       <widget class="QCheckBox" name="displayAttributesCheckBox">
        <property name="text">
         <string>Display attributes</string>
        </property>
       </widget>
      </item>

      <item row="6" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>

      <item row="5" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>

      <item row="7" column="0">
       <spacer name="verticalSpacer2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::SelectAnalyticsObjectTypesButton</class>
   <extends>QPushButton</extends>
   <header location="global">nx/vms/client/desktop/ui/common/select_analytics_object_types_button.h</header>
  </customwidget>
 </customwidgets>
<resources/>
<connections/>
</ui>
